packages/integrations/package.json

```diff
@@ -16,6 +16,7 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
+    "@sentry/apm": "5.15.5",
```

Comment:
Hmm this is interesting. How does it affect bundle size?
How does it impact users of the integrations that don't use `@sentry/apm`?

Reply:
It doesn't if you are not using Vue. It'll be fetched though.
I was thinking about passing apm's Tracing integration as a parameter, buuut not sure if it's worth it just yet.